org.feichtmeier.Musicpod.json

```diff
@@ -0,0 +1,228 @@
+{
+     "app-id": "org.feichtmeier.Musicpod",
+     "runtime": "org.freedesktop.Platform",
+     "runtime-version": "23.08",
+     "sdk": "org.freedesktop.Sdk",
+     "add-extensions": {
+          "org.freedesktop.Platform.ffmpeg-full": {
+               "directory": "lib/ffmpeg",
+               "version": "23.08",
+               "add-ld-path": "."
+          }
+     },
+     "cleanup-commands": [
+          "mkdir -p /app/lib/ffmpeg"
+     ],
+     "command": "musicpod",
+     "finish-args": [
+          "--share=ipc",
+          "--share=network",
+          "--socket=fallback-x11",
+          "--socket=wayland",
+          "--filesystem=home",
+          "--device=dri",
+          "--socket=pulseaudio",
+          "--env=TMPDIR=/var/tmp",
+          "--own-name=org.mpris.MediaPlayer2.musicpod.*",
```

Comment:
Flatpak automatically grants access to `org.mpris.MediaPlayer2.$appid`, if you can change this in the future you can drop this permission.

Reply:
OK, we'll keep that in mind.